playbooks/foreman_platform.yml

```diff
@@ -0,0 +1,13 @@
+- hosts: all
+  become: true
+  roles:
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - role: foreman_installer
+      scenario: katello
+      foreman_installer_options_internal_use_only: "--disable-system-checks --enable-foreman-plugin-discovery --enable-foreman-plugin-bootdisk --enable-foreman-plugin-remote-execution --enable-foreman-plugin-hooks --enable-foreman-plugin-openscap --enable-foreman-proxy-plugin-openscap --enable-foreman-proxy-plugin-remote-execution-ssh"
+      foreman_installer_additional_packages:
+          - katello
```

Comment:
It is indeed named like that, but the `katello` package is meta-package that pulls in all the dependencies too.  Makes the installer take less time since everything should already be on the box